tests/data/test-data.sql

```diff
@@ -301,18 +336,26 @@ insert into matchmaker_queue_game (matchmaker_queue_id, game_stats_id) values
 insert into map_pool (id, name) values
   (1, "Ladder1v1 season 1: 5-10k"),
   (2, "Ladder1v1 season 1: all"),
-  (3, "Large maps");
-
-insert into map_pool_map_version (map_pool_id, map_version_id) values
-  (1, 15), (1, 16), (1, 17),
-  (2, 11), (2, 14), (2, 15), (2, 16), (2, 17),
-  (3, 1),  (3, 2),  (3, 3);
+  (3, "Large maps"),
+  (4, "Generated Maps with Errors");
+
+insert into map_pool_map_version (map_pool_id, map_version_id, weight, map_params) values
+                                                                                       (1, 15, 1, NULL),(1, 16, 1, NULL),(1, 17, 1, NULL),
```

Comment:
...

Reply:
I think it tried to format it as python sigh